src/logic/calculate.js

```diff
@@ -0,0 +1,18 @@
+import operate from './operate';
+
+function calculate(calculator) {
+  let { total, next } = calculator;
```

Comment:
* [ ] Kindly revert to the old code. The function was not returning anything. all the previous reviewer sought to say is for you to return the mutated calculator object.